docs/docs-content/clusters/data-center/maas/create-manage-maas-clusters.md

```diff
@@ -65,6 +65,8 @@ To deploy a new MAAS cluster:
 
         - Resource Pool: The MAAS resource pool from which to select available servers for deployment. Filter available servers to only those that have at least the amount of CPU and Memory selected.
 
+    - Tags - You can specify tags to dynamically place nodes in a pool by using MAAS automatic tags. Specify the tag values that you want to apply to all nodes in the node pool. To learn more about MAAS automatic tags, refer to the [MAAS Tags](https://maas.cloud.cbh.kth.se/MAAS/docs/cli/how-to-tag-machines.html#heading--how-to-create-automatic-tags) documentation.
```

Comment:
```suggestion
    - Tags - The machine tags from which to select machines to dynamically place nodes in a pool. Specify the tag values that you want to meet criteria. To learn more about MAAS tags, refer to the [MAAS Tags](https://maas.io/docs/how-to-tag-machines) documentation.
```

@karl-cardenas-coding This feature is not about automatic tags creation by Palette. Palette will use existing MAAS tags already assigned on machines to dynamically assign machines, just like AZ or pool.